### ISSUE TITLE
[dst] Switching to new 2016 GCD file produces an extra Q and P frame (Trac #1892)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1884]
Switching the tests to use the 2016 GCD file produces an extra Q and P frame.  The tests check that only 10 frames each are seen.  I checked the GCD file and there are only three frames in it (G, C, and D), so it's not clear where the extra frame is coming from.

I broke this so it's mine...unless I can confirm this is an issue for Juan Carlos.

Migrated from https://code.icecube.wisc.edu/ticket/1892
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:13:24", 
    "description": "Switching the tests to use the 2016 GCD file produces an extra Q and P frame.  The tests check that only 10 frames each are seen.  I checked the GCD file and there are only three frames in it (G, C, and D), so it's not clear where the extra frame is coming from.\n\nI broke this so it's mine...unless I can confirm this is an issue for Juan Carlos.", 
    "reporter": "olivas", 
    "cc": "juancarlos", 
    "resolution": "fixed", 
    "_ts": "1550067204154158", 
    "component": "combo reconstruction", 
    "summary": "[dst] Switching to new 2016 GCD file produces an extra Q and P frame", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2016-10-13T18:06:21", 
    "milestone": "", 
    "owner": "juancarlos", 
    "type": "rumor / allegation"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20161111T19:11:23: 
* olivas changed owner from "olivas" to "juancarlos"
* olivas changed status from "new" to "assigned"

**Comment by icecube-trac** [comment]
Trac update at 20161115T18:05:41: juancarlos commented:

> This problem was that the number of frames issued included what is in GCD. Old GCD included and Info frame which the 2016 GCD does not. I changed the test generator to it handles the number of frames to generate.

**Comment by icecube-trac** [comment]
Trac update at 20161115T18:06:19: 
* juancarlos changed resolution from "" to "fixed"
* juancarlos commented:

> In changeset 151515:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="151515"
> Added configurable number of frames to generate. Closes #1884
> ```


* juancarlos changed status from "assigned" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:13:24: 
* olivas changed milestone from "October 2016" to ""
* olivas commented:

> Milestone October 2016 deleted
